tests/testthat/helper-mocks.R

```diff
@@ -0,0 +1,52 @@
+mock_cran_version <- function(version, .env = caller_env()) {
```

Comment:
We now have mocking helpers for mocks that come up in multiple places.

I think if you have a bunch of these, it probably does make sense to give them their own helper file.

I'm a bit ambivalent about some of these helpers that are super simple (i.e. fit on one line). Would it be better to have `local_mocked_binding()` calls in the tests themselves, in terms of "keeping the reader in the test"? OTOH if the helper has a good name, the reader probably doesn't feel the need to go read the definition.

Reply:
I think if they are used commonly I like having the helper functions if they are clearly and consistently named (see my comment below). I don't think it takes the reader out of the test as it's clearly obvious what they're doing.

Reply:
I also like having them in their own helper file, it makes discovering them much easier.

Reply:
I like them being in a helper function, but I don't love the names because they only tell you what function is being mocked, not what the mocked function will do. I think they also should have `local` in their name? 

Brainstorming:
* `local_cran_version()`
* `local_disable_install_check()`
* `local_pretend_is_package()`
* ...